shared/domains/chelonia/persistent-actions.js

```diff
@@ -231,7 +242,7 @@ sbp('sbp/selectors/register', {
   'chelonia.persistentActions/unload' (): void {
     for (const id in this.actionsByID) {
       // Clear the action's timeout, but don't cancel it so that it can later resumed.
-      this.actionsByID[id].timer && clearTimeout(this.actionsByID[id].timer)
+      this.actionsByID[id][timer] && clearTimeout(this.actionsByID[id].timer)
```

Comment:
Did you mean `clearTimeout(this.actionsByID[id][timer])`?

Reply:
I did